app/components/nolt-feedback-script.js

```diff
@@ -0,0 +1,13 @@
+/* eslint no-undef: "off" */
+import Component from '@ember/component';
+
+export default Component.extend({
+  didInsertElement() {
+    this._super(...arguments);
+
+    nolt('init', {
+      selector: '.nolt-button',
```

Comment:
Hmmm. Would it be better to create a component that [detects its own id](https://api.emberjs.com/ember/3.22/classes/Component/properties/elementId?anchor=elementId) and uses that to build a selector?